### PR TITLE
fix: typo fixes

### PR DESCRIPTION
**Pull request by tvolk131**

### CODE REVIEW

File: fedimint-core/src/fmt_utils.rs [hunk 1]
```diff
@@ -18,9 +18,9 @@ pub fn rust_log_full_enabled() -> bool {
 /// `RUST_LOG_FULL` env variable.
 ///
 /// Meant for logging errors.
-pub struct OptStracktrace<T>(pub T);
+pub struct OptStacktrace<T>(pub T);
```
Comment (dpc) [inline fedimint-core/src/fmt_utils.rs:21]: I know, I know. I was going to fix. :D

**Review by dpc — DISMISSED** [review]
Thanks!

**Comment by dpc** [comment]
@tvolk131 You missed a spot. :D

**Comment by tvolk131** [comment]
Hmm I double-checked for any uses of `OptStrackTrace` and didn't find any. I'm seeing that the CI fails due to the following error:

```
fedimint-workspace-build>    Compiling fedimint-rocksdb v0.1.0 (/build/source/fedimint-rocksdb)
fedimint-workspace-build> error[E0432]: unresolved import `fedimint_core::fmt_utils::OptStracktrace`
fedimint-workspace-build>   --> gateway/ln-gateway/src/lib.rs:42:5
fedimint-workspace-build>    |
fedimint-workspace-build> 42 | use fedimint_core::fmt_utils::OptStracktrace;
fedimint-workspace-build>    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^--------------
fedimint-workspace-build>    |     |                         |
fedimint-workspace-build>    |     |                         help: a similar name exists in the module: `OptStacktrace`
```

I don't see this line locally and I ran `nix build` locally and everything works fine - I'm not sure what I'm missing here. Apologies if this is something simple, I've read the docs but still can't figure out what's going on here.

**Comment by dpc** [comment]
@tvolk131

My guess: you need to rebase on top of latest master and then it will show. Someone (probably me, sorry ;D) added a new instance of it in last few days, and your PR is being tested a merge to master at tthe time you created PR, not at the time when you cloned the repo locally.

**Comment by tvolk131** [comment]
@dpc thanks! That makes sense - not sure why I didn't think of that. I rebased and amended that commit, everything should be good now!

**Review by dpc — APPROVED** [review]

**Pull request merged**